lib/resty/upstream/healthcheck.lua

```diff
@@ -201,8 +199,14 @@ local function peer_ok(ctx, is_backup, id, peer)
     end
 end
 
+-- shortcut error function for check_peer()
+local report_error = function(sock, ctx, is_backup, id, peer, ...) 
```

Comment:
Style: I think the form `local function report_error(...)` is more consistent with other local named functions in this module.